app.rb

```diff
@@ -0,0 +1,98 @@
+require "sinatra/base"
+require "sinatra/reloader"
+require_relative "lib/database_connection"
+require_relative "lib/peep_repository"
+require_relative "lib/peep"
+require_relative "lib/user_repository"
+require_relative "lib/user"
+require "date"
+
+DatabaseConnection.connect
+
+class Application < Sinatra::Base
+  enable :sessions
+
+  configure :development do
+    
+    register Sinatra::Reloader
+    also_reload "lib/peep.rb"
+    also_reload "lib/peep_repository.rb"
+    also_reload "lib/user.rb"
+    also_reload "lib/user_repository.rb"
+  end
+
+  get "/" do
+    erb(:index)
+  end
+
+  get '/feed' do
+    @peeps = PeepRepository.new.all.reverse
+    @user_repo = UserRepository.new
```

Comment:
What data will you specifically need from `@user_repo` in your view? It's usually better to "give" to the view the least access possible to any database resources, and only pass it the data it needs to display information to the user — in that case, maybe `@peeps` and a variable `@current_user` set in the route block would be enough, instead of letting the view have access to the whole repository class methods